app/models/order.rb

```diff
@@ -35,7 +39,15 @@ def set_total
   end
 
   def received
-    update(status: 'received', placed_on: Time.now)
+    update(status: :received, placed_on: Time.now)
+  end
+
+  def cancelled
```

Comment:
change method name. this looks like more like an state rather than cancelling method

Reply:
Done